stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js

```diff
@@ -109,7 +115,8 @@ const PROPS_TO_ALLOWED_VARS = {
   ...mapPropsToAllowedVars("line-height", FONT_LINE_HEIGHTS),
   ...mapPropsToAllowedVars("font-weight", FONT_WEIGHTS),
   ...mapPropsToAllowedVars("font-size", FONT_SIZES),
-  ...mapPropsToAllowedVars(TIMING_FUNCTION_PROPS, "--expand-animation-timing"),
+  ...mapPropsToAllowedVars(TIMING_FUNCTION_PROPS, ANIMATION_TIMING),
```

Comment:
`ANIMATION_TIMING` this is not defined - should be `ANIMATION_TIMINGS`

Reply:
@laviomri advised to correct the naming conventions, the const reference got away :)
Thanks

Reply:
we changed to "ANIMATION_TIMING" for naming conventions consistency